tests/test_initialize_simulation.py

```diff
@@ -12,40 +12,11 @@ class TestInitializeSimulation(TestCase):
     def test_initialize_simulation(self):
         conf = mock_config().suppress_logging()
 
-        sim, env = initialize_simulation(conf)
+        sim, env = initialize(conf)
         self.assertEqual(len(sim.vehicles), 20, "should have loaded 20 vehicles")
         self.assertEqual(len(sim.stations), 4, "should have loaded 4 stations")
         self.assertEqual(len(sim.bases), 2, "should have loaded 1 base")
 
-    def test_initialize_simulation_with_filtering(self):
```

Comment:
do we not want this test? i guess it's because the signature changed, but, couldn't these be turned into InitFunctions, tacked on at the end of the list:
```python
sim, env = initialize(
  conf, 
  init_functions=tuple(default_init_functions()) + (filter_veh, filter_base, filter_station)
)
```

Reply:
yeah, that's a good idea to just add the filter functions as init functions.